src/style.css

```diff
@@ -1 +1,107 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Book.ttf") format("truetype");
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Heavy.ttf") format("truetype");
+  font-weight: 800;
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-weight: 300;
+}
+
+/* #region bar */
+.search-bar {
+  display: flex;
+  align-items: center;
+  flex-shrink: 0;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  text-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  margin-top: 20px;
+  position: relative;
+  height: 70px;
+}
+
+.search-bar__small {
+  height: 42px;
+}
+
+.search-bar::after {
+  background-image: url(./images/Search.svg);
+  flex-shrink: 0;
+  background-size: contain;
+  content: "";
+  position: absolute;
+  height: 19px;
+  width: 19px;
+  left: 26px;
+}
+
+.search-bar__small::after {
+  height: 11px;
+  width: 11px;
+  left: 13px;
+}
+
+.search-bar:hover {
+  background: #FFF;
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.20);
+}
+
+.search-bar:focus {
+  border-radius: 3px;
+  background: linear-gradient(180deg, #FFF 0%, #F6F6F7 100%);
+}
+
+.search-bar input[type="text"] {
+  border: none;
+  outline: none;
+}
+
+/* #endregion */
+
+/* #region text */
```

Comment:
Remove comments everywhere
```suggestion
```

Reply:
Could you please explain why? In lecture videos they are used almost everywhere and it really helps

Reply:
yes, it could be helpful during the development (when you write code), but it's not ok to push it (in case this code goes to production)
Generally, it's not critical here, but at least FYI - we don't want to push such code in production in real life

In the future you will divide a separate blocks in separate files anyway :slightly_smiling_face:

Reply:
thank you foR the explanation!